Sidenav: test continues to be extremely flaky

**Expected behavior**

Tests pass reliably

**Actual behavior**

This test continues to fail on first run of most pull requests, requiring rerunning of the action or new commits to be pushed to kick off the process again:

```
lib/components/sidenav/pharos-sidenav.test.js:

 ❌ pharos-sidenav > resets its slide status when going back to a viewport above 1055px (failed on Webkit)
      AssertionError: expected true to be false
      + expected - actual
      
      -true
      +false
      
      at r (../../node_modules/@esm-bundle/chai/node_modules/.pnpm/chai@4.3.4/node_modules/chai/chai.js:9575:6)
      at ../../node_modules/@esm-bundle/chai/node_modules/.pnpm/chai@4.3.4/node_modules/chai/chai.js:251:8
      at ../../node_modules/@esm-bundle/chai/node_modules/.pnpm/chai@4.3.4/node_modules/chai/chai.js:1072:9
      at e (../../node_modules/@esm-bundle/chai/node_modules/.pnpm/chai@4.3.4/node_modules/chai/chai.js:7959:33)
      at lib/components/sidenav/pharos-sidenav.test.js:54:38
```

**Additional information**

An attempt was made to fix this in #535 but it seems only to have delayed rather than fixed the issue summarily. In the worst case, we can decide whether this test is providing value and remove it if we can't find a way to make it more reliable.

I became quite interested in this issue when I noticed the inconsistency of this bug, so I went in search of reasons for its existence. However, the error occurred here in Firefox, not in Webkit.

### Error


```
lib/components/sidenav/pharos-sidenav.test.js:

 ❌ pharos-sidenav > resets its slide status when going back to a viewport above 1055px (failed on Firefox)
      AssertionError: expected true to be false
      + expected - actual
      
      -true
      +false
      
      at r (../../node_modules/@esm-bundle/chai/node_modules/.pnpm/chai@4.3.4/node_modules/chai/chai.js:9574:12)
      at n.exports<[3]</t.exports/i.prototype.assert (../../node_modules/@esm-bundle/chai/node_modules/.pnpm/chai@4.3.4/node_modules/chai/chai.js:250:12)
      at n.exports<[5]</t.exports/< (../../node_modules/@esm-bundle/chai/node_modules/.pnpm/chai@4.3.4/node_modules/chai/chai.js:1072:9)
      at e (../../node_modules/@esm-bundle/chai/node_modules/.pnpm/chai@4.3.4/node_modules/chai/chai.js:7959:28)
      at e (../../node_modules/@esm-bundle/chai/node_modules/.pnpm/chai@4.3.4/node_modules/chai/chai.js:9372:21)
      at lib/components/sidenav/pharos-sidenav.test.js:55:15
``` 

###  Possible causes:

- Firefox calculates pixels in floating-point values, unlike others that use integers. This can lead to inconsistencies in sizes, spaces, etc., but I haven't studied much about it.

- According to (https://css-tricks.com/forums/topic/firefox-ignoring-max-width320px-media-query/), the devtools in Firefox could interfere with the viewport, which only confused me further.

However, while trying to gather more information about what was happening, I added this line of code to the test: `window.getComputedStyle(component, null).getPropertyValue('width')`, and it simply started working correctly. I apologize for not being able to bring you solutions, but maybe we can start thinking about one.

--- a/packages/pharos/src/components/sidenav/pharos-sidenav.test.ts
+++ b/packages/pharos/src/components/sidenav/pharos-sidenav.test.ts
@@ -129,10 +129,11 @@ describe('pharos-sidenav', () => {
     expect(document.activeElement === renderedButton).to.be.true;
   });
 
-  it('resets its slide status when going back to a viewport above 1055px', async () => {
+  it.only('resets its slide status when going back to a viewport above 1055px', async () => {
     await setViewport({ width: 1055, height: 768 });
     component.slide = true;
     await component.updateComplete;
+    await aTimeout(1000);
 
     await setViewport({ width: 1056, height: 768 });
     await component.updateComplete;

--- a/packages/pharos/src/components/sidenav/pharos-sidenav.test.ts
+++ b/packages/pharos/src/components/sidenav/pharos-sidenav.test.ts
@@ -129,7 +129,7 @@ describe('pharos-sidenav', () => {
     expect(document.activeElement === renderedButton).to.be.true;
   });
 
-  it.only('resets its slide status when going back to a viewport above 1055px', async () => {
+  it('resets its slide status when going back to a viewport above 1055px', async () => {
     await setViewport({ width: 1055, height: 768 });
     component.slide = true;
     await component.updateComplete;

